viewmaster/pdsgrouptable.py

```
@@ -250,13 +446,28 @@ def sort_tables(tables):
     def tables_from_pdsfiles(pdsfiles, exclusions=set(), hidden=set(),
```

Ruff (0.15.5)

[warning] Missing return type annotation for staticmethod `tables_from_pdsfiles`

(ANN205)